File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/cathedries/CathedriesMvpView.java
package ru.ulstu_team.ulstuschedule.ui.cathedries;

import java.util.List;

import ru.ulstu_team.ulstuschedule.data.model.Cathedra;
import ru.ulstu_team.ulstuschedule.ui.base.MvpView;

public interface CathedriesMvpView extends MvpView {

    void showCathedries(List<Cathedra> cathedries);

    void showEmptyList();

    void showError();

}

File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/cathedries/CathedriesMvpView.kt
package ru.ulstu_team.ulstuschedule.ui.cathedries

import ru.ulstu_team.ulstuschedule.data.model.Cathedra
import ru.ulstu_team.ulstuschedule.ui.base.MvpView

interface CathedriesMvpView : MvpView {

    fun showCathedries(cathedries: List<Cathedra>)

    fun showEmptyList()

    fun showError()

}
